### PR TITLE
[Homebrew Doc] Contributing Guidelines: Mention the bot merging PRs

### PR DESCRIPTION
**Pull request by AliSoftware**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] ~~Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?~~
- [ ] ~~Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?~~
- [ ] ~~Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?~~

-----

This is just to mention that once approved, the PRs will be auto-merged by the bot. This way contributors won't wonder why the person who approved their PR didn't merge it immediately after approval 👍 

\cc @SMillerDev

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ First time contributing to Homebrew? Read our [Code of Conduct](https://github.c
 * `git commit` with message formatted `foo 2.3.4 (new formula)`
 * [open a pull request](https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request) and fix any failing tests
 
+Once you've addressed any potential feedback and a Member of the Homebrew org has approved your Pull Request, the [BrewTestBot](https://github.com/BrewTestBot) will automatically merge it a couple of minutes later.
```
Comment (AliSoftware) [inline CONTRIBUTING.md:25]:
```suggestion
Once you've addressed any potential feedback and a member of the Homebrew org has approved your pull request, the [BrewTestBot](https://github.com/BrewTestBot) will automatically merge it a couple of minutes later.
```

**Comment by SMillerDev** [comment]
I think this might be better suited for docs.brew.sh, which lives over in Homebrew/brew.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -22,6 +22,8 @@ First time contributing to Homebrew? Read our [Code of Conduct](https://github.c
 * `git commit` with message formatted `foo 2.3.4 (new formula)`
 * [open a pull request](https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request) and fix any failing tests
 
+Once you've addressed any potential feedback and a member of the Homebrew org has approved your pull request, the [BrewTestBot](https://github.com/BrewTestBot) will automatically merge it a couple of minutes later.
```
Comment (SMillerDev) [inline CONTRIBUTING.md:25]: It's not exclusively testbot by the way, it can still be manually merged by maintainers.

**Comment by AliSoftware** [comment]
> I think this might be better suited for docs.brew.sh, which lives over in Homebrew/brew.

I wondered about it too, but tbh since when you open a PR the link in the PR template points to the Contributing Guidelines, that's where I looked first. So I think it could be worth mentioning it in moth places?

File: CONTRIBUTING.md (hunk 2 above)
Comment (AliSoftware) [reply CONTRIBUTING.md:25]: What about this wording?
```suggestion
Once you've addressed any potential feedback and a member of the Homebrew org has approved your pull request, your PR will be automatically merged by our [@BrewTestBot](https://github.com/BrewTestBot) a couple of minutes later – if not merged by a maintainer manually before.
```

Comment (SMillerDev) [reply CONTRIBUTING.md:25]: I think I'd prefer 

> Once you've addressed any potential feedback and a member of the Homebrew org has approved your pull request, your PR will be automatically merged by our [@BrewTestBot](https://github.com/BrewTestBot), or a maintainer.

**Review by SMillerDev — APPROVED** [review]
Thanks @AliSoftware ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Pull request closed without merging**